frontend/src/modules/machinery/components/TranslationSource.js

```diff
@@ -14,53 +15,52 @@ import TranslationMemory from './TranslationMemory';
 
 type Props = {|
     translation: MachineryTranslation,
+    locale: Locale,
 |};
 
 
 /**
  * Shows a list of translation sources.
  */
-export default function TranslationSource(props: Props) {
-        const { translation } = props;
+export default function TranslationSource({ translation, locale }: Props) {
+    const translationSource =
+        translation.sources.map((source, index) => {
```

Comment:
Please move this to the previous line (after the `=` sign), and then dedent the whole `switch` block by one level.